### ISSUE TITLE
"balena tunnel" offers selection menu despite given UUID

### ISSUE THREAD

**Issue by pdcastro** [balena-io/balena-cli#1287]
It shouldn't ask to select the device as follows:

```
$ ./bin/balena tunnel <either full or short UUID> -p 22222:4321
[Info]    Tunnel to b1878c6
? Select a device (Use arrow keys)
❯ silent-thunder (b1878c6)
  ...
```

- **Cli version:** tested with 10.13.5 and 11.0.0
- **Standalone install:** no
- **Node version:** 8.16.0
- **Npm version:** 6.4.1
- **Operating system and architecture:** macOS 64-bit

**Comment by pdcastro** [comment]
This sounds like a _different_ case than the one fixed by @wrboyce in this PR:
[tunnel: allow using partial device uuids](https://github.com/balena-io/balena-cli/pull/1180/files)

And presumably it is something that started happening after that PR was merged -- which was in CLI v10.4.0. There were SDK changes...

**Comment by pdcastro** [comment]
This issue has been fixed. Not sure exactly which CLI version fixed it, possibly **v11.3.4** (PR #1324) as it includes a commit of subject _"Tunnel: Fix incorrect parsing of numeric short UUIDs"._ I have tested it now with **v11.18.3** and the issue no longer happens (the interactive selection menu is not displayed, which is correct).
